Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/Cell/GiphyCollectionCell.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  CustomCell.swift
+//  StreamChat
+//
+//  Created by Mohammed Hanif on 27/04/22.
+//  Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
```

Comment:
@mdhanif-simformsolutions  Please remove unnecessary imports

Reply:
Updated this. thank you.